.vscode/extensions.json

```diff
@@ -5,7 +5,6 @@
   // List of extensions which should be recommended for users of this workspace.
   "recommendations": [
     "ms-edgedevtools.vscode-edge-devtools",
-    "msoffice.microsoft-office-add-in-debugger",
```

Comment:
Just a note that while this extension is deprecated it is needed when debugging older version of Office that don't support webview2.  The documentation would need to be updated to tell developers to install this depricated extension for those versions of office.

Reply:
Thanks for notice. @Rick-Kirkham  FYI.

Reply:
I have created a work item for this and tentatively scheduled it for April.